app/views/fact_check_content_review.scala.html

```diff
@@ -43,7 +43,9 @@ <h2 class="app-task-list__section">
         @for((page, index) <- review.pages.zipWithIndex) {
           <li class="app-task-list__item">
             <span class="app-task-list__task-name">
-              <a href="@{routes.FactCheckPageReviewController.onPageLoad(review.ocelotId, page.title.drop(1)).toString}" aria-describedby="page-@index">@page.title</a>
+              <a href="@{routes.FactCheckPageReviewController.onPageLoad(review.ocelotId, page.title.drop(1)).toString}" aria-describedby="page-@index">
+                  <span class="govuk-visually-hidden">@messages("factCheck.visuallyHiddenPage") @page.title</span>@page.title
+              </a>
```

Comment:
The additional text on the link causes the journey tests to fail, so you will have to chat to Roheel to workout whatto change

Reply:
Yes, they caused problems for the unit tests also. The hidden text seems to be included in link text when you access the link.